monkey/agent_plugins/exploiters/ssh/config-schema.json

```
@@ -0,0 +1,15 @@
+{
+    "required": [
+        "ssh_connect_timeout"
+    ],
+    "properties": {
+        "ssh_connect_timeout": {
```

What about
```suggestion
        "connection_timeout": {
```

`ssh` is probably redundant.